bin/newman.js

```diff
@@ -28,6 +29,8 @@ program
     .option('--folder <path>',
         'Specify the folder to run from a collection. Can be specified multiple times to run multiple folders',
         util.cast.memoize, [])
+    .option('--item-from-folder <path>', 'Specify the item and its parent folder',
```

Comment:
Use `request` vocabulary instead of item please. 😊

Also the parameter name is not sitting well. Postman runner has a way to sub-select which requests to run and this should feel close to that.